JamGame/Assets/Scripts/Level/DailyBill/Model.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Common;
+using System.Linq;
+using TileBuilder;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Level.DailyBill
+{
+    [SerializeField]
+    public struct Check
+    {
+        public int Rent;
+        public int Sum => Rent;
+    }
+
+    [AddComponentMenu("Scripts/Level.DailyBill.Model")]
+    public class Model : MonoBehaviour
+    {
+        [SerializeField]
+        private TileBuilderImpl tileBuilder;
+
+        [SerializeField, InspectorReadOnly]
+        private Check check;
+        public Check Check => check;
+
+        public UnityEvent<Check> CheckChanged;
+
+        public void UpdateCheck()
```

Comment:
You can compute the check inside getter and request it from model only when view becomes active(as it definiteley won't change when the view is opened)

Reply:
But better introduce new method `ComputeCheck` instead of getter